userguide/services.rst

```diff
@@ -927,40 +926,36 @@ module.
 
    +------------------------+-------------------+--------------------------------------------------------------------------+
    | Setting                | Value             | Description                                                              |
-   |                        |                   |                                                                          |
    +========================+===================+==========================================================================+
-   | Name                   | string            | Mandatory. This is required to match the setting on the rsync client.    |
-   |                        |                   |                                                                          |
+   | Name                   | string            | This **must** match the settings on the rsync client.                    |
```

Comment:
Improved, but still not clear how a name can match the settings on the client.  Which settings?  Is this a username?